tests/cpp/test_matmul_scheduler.cpp

```diff
@@ -3522,22 +3528,24 @@ TEST_P(HopperMatmulSchedulerTest, FusedMultiplySumBiasNeg) {
 
 INSTANTIATE_TEST_SUITE_P(
     General,
-    HopperMatmulSchedulerTest,
+    HopperPlusMatmulSchedulerTest,
     testing::Combine(
         testing::Bool(), // use_smem_epilogue
         testing::Bool(), // a_k_inner
         testing::Bool(), // b_k_inner
         testing::Values(512), // M
         testing::Values(256), // N
         testing::Values(128), // K
-        testing::Values(MmaMacro::Hopper_64_128_16), // mma_macros
+        testing::Values(
+            MmaMacro::Hopper_64_128_16,
+            MmaMacro::Blackwell_128_128_16), // mma_macros
```

Comment:
```suggestion
            MmaMacro::Blackwell1CTA_128_128_16), // mma_macros
```
**[Suggested by AI]** Verified that the build will pass with this change.